aiida_quantumespresso/cli/calculations/dos.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""Command line scripts to launch a `PwCalculation` for testing and demonstration purposes."""
+from __future__ import absolute_import
+import click
+
+from aiida.orm import RemoteData
+from aiida.cmdline.params import options, types
+from aiida.cmdline.utils import decorators
+
+from aiida_quantumespresso.cli.utils import options as options_qe
+
+# required inputs: code, input_parameters, (parent_calculation or parent_folder)
+# optional: settings
+
+
+@click.command()
+@options.CODE(required=True, type=types.CodeParamType(entry_point='quantumespresso.dos'))
+# Note: you cannot ask for a CalcJobNode created by a specific plugin, because you can get subclasses of Process via the
+# entry point system, but not subclasses of Node. You could filter by process_type of the CalcJobNode, but this is
+# currently not implemented in IdentifierParamType.
+#@options.CALCULATION(required=True, type=types.CalculationParamType(
```

Comment:
Instead of commented code, please put this in an issue to add and reference an issue on `aiida-core` for the feature request.